app/app/settings.py

```diff
@@ -36,6 +36,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'transactions',
```

Comment:
- Added a new app for all transactions logic. adding it to the list of our installed apps